src/components/Loans/OpenLoans/OpenLoansControl.js

```diff
@@ -253,9 +257,9 @@ class OpenLoansControl extends React.Component {
     }
   };
 
-  feeFineCount = (loan) => {
+  feeFineCount = loan => {
     const { resources } = this.props;
-    const accounts = get(resources, ['loanAccount', 'records'], []);
+    const accounts = resources?.loanAccount?.records || [];
```

Comment:
Thanks for syntax updates. It looks great.
Please, provide a refactoring in the separated commit to separate functional and styling changes. It might be useful for reviewers.